products/filemaker.md

```diff
@@ -7,6 +7,11 @@ releaseColumn: false
 eolColumn: Support
 
 releases:
+-   releaseCycle: "20"
```

Comment:
As explained here, the new release is 2023, not 20: https://www.claris.com/blog/2023/get-ready-for-the-newest-claris-filemaker-version-filemaker-2023

```suggestion
-   releaseCycle: "2023"
```